### PR TITLE
[JENKINS-64341] Fix for tables-to-divs

### PR DESCRIPTION
**Pull request by dariver**
Jira: [JENKINS-64341](https://issues.jenkins-ci.org/browse/JENKINS-64341)

### What has been done
1. Fix for tables-to-div regression

### Screenshots

Before | After
:-: | :-:
before|after |

### How to test
1. Create a Freestyle job, add a new Post-Build action and fill the values
2. Save and verify that all the values were persisted.

### Checklist

- [ ] Git commits follow [best practices](https://chris.beams.io/posts/git-commit/) 
- [ ] Build passes in Jenkins 
- [ ] Appropriate tests or explanation to why this change has no tests 
- [ ] Pull Request is marked with appropriate label (see `.github/release-drafter.yml`) 
- [ ] JIRA issue is well described (problem explanation, steps to reproduce, screenshots) 
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### CODE REVIEW

**Comment by dariver** [comment]
cc @timja, @fqueiruga, @fbelzunc

**Review by timja — APPROVED** [review]
I had to combine it with https://github.com/jenkinsci/plot-plugin/pull/63 to test with: 

```
 mvn hpi:run -Djenkins.version=2.280 -Djava.awt.headless=true -Dport=8085
```

File: src/main/resources/lib/plot/blockWrapper.jelly
Comment (timja) [inline src/main/resources/lib/plot/blockWrapper.jelly:6]: why not just remove the whole file and use blockWrapper?

File: src/main/resources/lib/plot/blockWrapperBackground.jelly
Comment (dariver) [reply src/main/resources/lib/plot/blockWrapperBackground.jelly:2]: I've removed the div background. Keeping `blockWrapperBackground.jelly` so the tables in the old version remains still have the background color

File: src/main/resources/lib/plot/blockWrapper.jelly
Comment (dariver) [reply src/main/resources/lib/plot/blockWrapper.jelly:6]: Check my previous comment

File: src/main/resources/lib/plot/blockWrapperBackground.jelly
Comment (timja) [reply src/main/resources/lib/plot/blockWrapperBackground.jelly:2]: any reason for this?

Comment (dariver) [reply src/main/resources/lib/plot/blockWrapperBackground.jelly:2]: It's just that I didn't want to modify the original behavior with tables. But If you think that there is not problem with that I can remove `blockWrapperBackground.jelly` and reuse `blockWrapper.jelly`. But we will lose the gray background in Jenkins old versions.

Comment (timja) [reply src/main/resources/lib/plot/blockWrapperBackground.jelly:2]: I would remove it, but not a blocker and not a maintainer here

**Review by fqueiruga — APPROVED** [review]

**Comment by mkrauser** [comment]
I can't wait for this to get merged

**Comment by vgaidarji** [comment]
Thank you all for the contribution and reviews here. I'm on very limited availability last weeks so not even sure when I could provide support here. For those reasons I've added a `adopt-this-plugin` label to the repo a while ago.
@timja is a good candidate to become the next maintainer of this awesome plugin. I've reached to him directly to confirm whether he's open to such opportunity.

**Pull request merged**
